packages/zipkin-instrumentation-koa/README.md

```diff
@@ -0,0 +1,20 @@
+# zipkin-instrumentation-koa
+
+Zipkin middleware for Koa 2.x that traces incoming HTTP requests.
+
+## Installation
+`npm install zipkin-instrumentation-koa`
+
+## Usage
+```js
+const Koa = require('koa');
+const {Tracer, ConsoleRecorder, ExplicitContext} = require('zipkin');
+const {zipkinMiddleware} = require('zipkin-instrumentation-koa');
+
+const recorder = new ConsoleRecorder();
+const ctxImpl = new ExplicitContext();
+const tracer = new Tracer({recorder, ctxImpl});
+const app = new Koa();
+
+app.use(zipkinMiddleware({tracer, serviceName: 'zipkin-koa-demo', port: 3000}));
```

Comment:
please remove the service name arg for configuring that in the tracer. this helps keep trace data consistent. you can look in other README files for an example.